src/App.js

```diff
@@ -1,19 +1,49 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import './App.css';
+
+
 
 const App = () => {
+
+  const [entries, setEntries] = useState(chatMessages)
+  const toggleLike = (id) => {
```

Comment:
I know that we did together in my office hours, but please reach out if you need any further explanations as to **how and why** we want to do this. Great job on the implementation!